src/collatzConjecture.js

```diff
@@ -0,0 +1,15 @@
+export default function collatzConjecture(n){
+  let collatzArr = []
+  collatzArr.push(n)
+  while(n > 1){
```

Comment:
Consider the use of a local variable to avoid overwriting the input parameter `n`